clients/java/signalr/src/main/java/com/microsoft/signalr/InvocationBindingFailureMessage.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+package com.microsoft.signalr;
+
+class InvocationBindingFailureMessage extends HubMessage {
+    private final String invocationId;
```

Comment:
In the other message types we include the HubMessage type as a property 
So something `private final int type = HubMessageType.INVOCATION_BINDING_FAILURE`

Reply:
The reason we have that is so that gson writes the `type` field into the json. Since this type never goes through gson it doesn't need it.